src/App/Fossa/Test.hs

```diff
@@ -119,3 +125,15 @@ testMain config = do
 
     renderJson :: (Has (Lift IO) sig m, Has Logger sig m) => Issues -> m ()
     renderJson = logStdout . decodeUtf8 . Aeson.encode
+
+    -- `issueDashURL` will have the branch set to `master` for all issues. The url will contain: `/refs/branch/master`
+    -- Update `issueDashURL` for all issues with the current revision's branch.
```

Comment:
Can you add a specific example of before and after? I think this is close, but just write out something like `/refs/branch/differentbranch`